cmd/cluster-agent-cloudfoundry/app/app.go

```diff
@@ -201,6 +202,14 @@ func run(cmd *cobra.Command, args []string) error {
 		return log.Errorf("Error while starting agent API, exiting: %v", err)
 	}
 
+	if config.Datadog.GetBool("cluster_agent.admission_controller.enabled") {
+		log.Info("Starting admission controller server")
+
+		if err = admissioncontroller.StartServer(); err != nil {
+			return log.Errorf("Error while starting admission controller server, exiting: %v", err)
```

Comment:
We don't want to return the error that is returned by the logger. In most cases it will be `nil`. We should log, and then return the error.
```suggestion
			log.Errorf("Error while starting admission controller server, exiting: %v", err)
			return err
```